src/views/Pools/components/Apr.tsx

```diff
@@ -59,22 +64,31 @@ const Apr: React.FC<AprProps> = ({ pool, showIcon, stakedBalance, performanceFee
     onPresentApyModal()
   }
 
+  // eslint-disable-next-line no-restricted-globals
+  const isValidate = apr !== undefined && !isNaN(apr)
```

Comment:
You can use Number.isNaN therefore eslint can be removed

Reply:
@memoyil  updated

Reply:
👍